cmake/Dependencies.cmake

```diff
@@ -84,17 +84,14 @@ endif()
 
 # ---[ On Android, Caffe2 uses cpufeatures library in the thread pool
 if (ANDROID)
-  # ---[ Check if cpufeatures was already imported
   if (NOT TARGET cpufeatures)
     add_library(cpufeatures STATIC
-      "${ANDROID_NDK}/sources/android/cpufeatures/cpu-features.c")
+        "${ANDROID_NDK}/sources/android/cpufeatures/cpu-features.c")
     target_include_directories(cpufeatures
-      PUBLIC "${ANDROID_NDK}/sources/android/cpufeatures")
-    target_link_libraries(cpufeatures PRIVATE dl)
+        PUBLIC "${ANDROID_NDK}/sources/android/cpufeatures")
+    target_link_libraries(cpufeatures PUBLIC dl)
```

Comment:
No need to change to `PUBLIC`, cpufeatures interface is not exposed even if you rely on Caffe2 internals

Reply:
Actually for static lib it will be public anyway - as the linkage flags are transitive for static libs regardless of its setting. I put a “public” here to explicitly note this :)